trade-mapping-to-ipaffs/src/main/java/uk/gov/defra/stw/mapping/toipaffs/common/MeansOfTransportFromEntryPointMapper.java

```diff
@@ -1,69 +1,42 @@
 package uk.gov.defra.stw.mapping.toipaffs.common;
 
 import java.util.List;
-import java.util.Map;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
-import org.springframework.util.CollectionUtils;
 import uk.gov.defra.stw.mapping.dto.MainCarriageSpsTransportMovement;
 import uk.gov.defra.stw.mapping.dto.SpsCertificate;
 import uk.gov.defra.stw.mapping.toipaffs.Mapper;
-import uk.gov.defra.stw.mapping.toipaffs.exceptions.NotificationMapperException;
+import uk.gov.defra.stw.mapping.toipaffs.common.utils.MeansOfTransportUtil;
 import uk.gov.defra.tracesx.notificationschema.representation.MeansOfTransportBeforeBip;
-import uk.gov.defra.tracesx.notificationschema.representation.enumeration.TransportMethod;
 
 @Component
 public class MeansOfTransportFromEntryPointMapper implements
```

Comment:
Refactored the means of transport to/from BCP mappers to simplify and reuse across all the certificate types (as they are the same but unnecessarily had a separate mapper for each type) and across the to/from mappers as they also are quite similar.